test/package.json

```diff
@@ -35,8 +41,6 @@
 		"node": ">=10"
 	},
 	"scripts": {
-		"test:run": "wdio ./suites/$SUITE/$SUITE.conf.ts",
-		"test:run-filter": "wdio ./suites/$SUITE/$SUITE.conf.ts --spec $FILTER",
 		"lint": "npx eslint . --ext .ts",
 		"fix": "npx eslint . --fix --ext .ts"
```

Comment:
Should include the `.js` file added

Reply:
Actually I just made another pass at TS'ing this file and after doing less with `yargs` than I was initially this went well. Now `cli.ts` with a bit of clean-up in there to make it a bit clearer.